assets/sass/components/overlay-notification/_googlesitekit-overlay-notification.scss

```diff
@@ -28,20 +30,30 @@
 	 * @see https://github.com/google/site-kit-wp/issues/8414
 	 */
 	@media (min-width: $bp-tablet) {
-		bottom: 36px;
+		bottom: 32px;
 		box-shadow: 0 8px $grid-gap-desktop 0 rgba(0, 0, 0, 0.25);
-		inset-inline-end: 36px;
+		inset-inline-end: 32px;
```

Comment:
Hi @kuasha420, I'm not actually seeing the `inset-inline-end` declaration in the resulting CSS.

Not sure if I am missing something but I get the impression our build toolchain doesn't support it yet.

![image](https://github.com/google/site-kit-wp/assets/18395600/648a7330-515f-4c23-8623-290b99ba3dbd)

Please can you check this, and remove the use of the declaration if it's not doing anything?

Reply:
Hi @techanvil, 

That is correct, our version of postcss transpiles logical properties to be regular left/right depending on the `dir` property of html element. We have a issue for upgrading our infrastructure to non do this ans use logical-properties, among other widely supported CSS features [here](https://github.com/google/site-kit-wp/issues/8414). 

I've kept the logical property here and added a comment in the top of this block (not visible in the diff) thar links the aforementioned issue. My thinking here is that, this is really not `right` because it can be left and right depending on text direction. This makes is easy to reason about and the workaround will be removed once we get around to modernizing our infra. 

Hopefully that clears it up, I can remove this if you want, but I'm inclined to keep it to hammer it home that this overlay should be placed on the `logcal-end`, not right or left. 

Cheers.

Reply:
Hi @kuasha420, thanks for reminding me about that.

Leaving `inset-inline-end` in for this reasons seems fair, but I don't think the comment is clear enough to help us remember that specific CSS property being related to it. Please can you update the comment to include a mention of `inset-inline-end` so we don't run into the same question again?